account_invoice_mgmt_dms/views/account_journal_views.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_account_journal_form" model="ir.ui.view">
+        <field name="name">account.journal.form.inherit</field>
+        <field name="model">account.journal</field>
+        <field name="inherit_id" ref="account.view_account_journal_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='advanced_settings']/group">
+                <group string="Documental Management">
+                    <field name="skip_invoice_publish_validator"/>
```

Comment:
Edition for this field is open for every user that can actually modify an `account.journal` record. Could you check which groups are currently able to do this?

Reply:
![imagen](https://github.com/user-attachments/assets/9acd1d78-0bc9-4f6b-a463-14cd5d3d9722)